pypit/arsciexp.py

```diff
@@ -466,6 +471,88 @@ def MasterFlatField(self, fitsdict, det):
         self.SetMasterFrame(mspixelflatnrm, "normpixelflat", det)
         return True
 
+    def MasterSlitFlat(self, fitsdict, det):
+        """
+        Generate Master slit flat frame for a given detector
+
+        Parameters
+        ----------
+        fitsdict : dict
+          Contains relevant information from fits header files
+        det : int
+          Index of the detector
+
+        Returns
+        -------
+        boolean : bool
+          Should other ScienceExposure classes be updated?
+        """
+        dnum = settings.get_dnum(det)
+        # If the master edge is already made, use it
+        if self._msslitflat[det - 1] is not None:
+            msgs.info("An identical master slit frame already exists")
+            return False
+        if settings.argflag['reduce']['slitedge']['useframe'] in ['trace', 'slitflat']:
```

Comment:
Now I am a bit confused.  What is the difference between a trace flat and a slit flat?